var/spack/repos/builtin/packages/kokkos-kernels/package.py

```diff
@@ -136,19 +136,21 @@ class KokkosKernels(CMakePackage, CudaPackage):
 
     tpls = {
         # variant name   #deflt   #spack name   #root var name #docstring
```

Comment:
Do you want to add the condition to this line so it matches the change?

Reply:
Yes, update the comment to add the condition up there